wiki/Ranking_Criteria/osu!catch/en.md

```diff
@@ -37,21 +38,20 @@ Overall rules and guidelines apply to every kind of osu!catch difficulty.
 
 #### Rules
 
-- **Your map must theoretically be possible to SS.** This means it must be possible to catch absolutely all fruits, including droplets.
-- **Edge dashes must not be used in direct conjunction with hyperjumps.** This is because such patterns require especially precise movement and force an unreasonable restriction on accuracy required to catch them.
-- **Each map must use at least two different combo colours which must not blend in with the map's background/storyboard/video.** This is so hit objects are always visible to the player.
+- **Your map must theoretically be possible to SS.** This means it must be possible to catch absolutely all fruits, drops and droplets.
+- **Each map must use at least two different combo colours which must not blend in with the map's background/storyboard/video.** This is so objects are always visible to the player.
 - **Do not use keysounds without hitnormal support.** If sounds blend perfectly with the song, feedback for hitting notes is minimal.
 
 #### Guidelines
 
 - **All circles and slider heads should represent a sound existing in the music.** This is usually a distinct sound, but can also represent a continuous sound with an indistinguishable start or end.
+- **Sliderends of extended sliders should be snapped according to the song's beat structure.** If the song is using a straight beat, 1/4, 1/8, and 1/16 should be used. If the song is using a swing beat, 1/6 or 1/12 should be used. If the song has a sound in a different snap from what was recommended, snapping to an actual beat always takes priority.
+- **Dashes and hyperdashes should not be used when the destination of the dash or hyperdash is located near the left or right border of the play field.** This creates an uncomfortable movement as the catcher is forcibly stopped upon reaching the border of the playfield. Try to leave at least 16 osu!pixels of space between the end point of the dash/hyperdash and the border of the play field, namely at x:16 or x:496 at most.
```

Comment:
```suggestion
- **Dashes and hyperdashes should not be used when the destination of the dash or hyperdash is located near the left or right border of the playfield.** This creates an uncomfortable movement as the catcher is forcibly stopped upon reaching the border of the playfield. Leave at least 16 osu!pixels of space between the end point of the dash/hyperdash and the border of the playfield.
```

Reply:
It's important that x:16 and x:492 are left in, since those are the limits where the catcher gets stopped by the wall

Reply:
that's already said in different words in the previous sentence, and I think it's more relevant when put that way to the point u want to make (this is just preference in the end though, either way is probably fine)

Reply:
reworded